sig-cloud-provider/README.md

```diff
@@ -48,6 +48,10 @@ The following subprojects are owned by sig-cloud-provider:
 - **cloud-provider-vsphere**
   - Owners:
     - https://raw.githubusercontent.com/kubernetes/cloud-provider-vsphere/master/OWNERS
+- **cloud-provider-extraction**
+  - Owners:
```

Comment:
Are @cheftako and @d-nishi proposed as OWNERS? Reasonable to add this now?

Reply:
Yeah this needs at least one OWNERS file linked

Reply:
So this is part of the concern I had raised in https://github.com/kubernetes/community/pull/2731, we don't currently have an OWNERS file because this effort is about migrating/removing code so there is no offical code ownership even though we need to meet weekly.